frontend/src/client/Client.ts

```diff
@@ -0,0 +1,8 @@
+import type { ClientInterface } from './ClientInterface';
+import getResult from './api/GetResult/implement';
+
+const Client: ClientInterface = {
+  getResult,
+};
+
+export default Client;
```

Comment:
Just asking if are there any reason making it pascal case for ApiContext, Client and ClientInterface  (both in file name and variable name)?